documentation/modules/rn-2.5.adoc

```diff
@@ -241,57 +248,57 @@ spec:
 
 This issue is resolved in {project-short} {project-version}, the new forms of create and update provider display up-to-date properties of the provider. link:https://issues.redhat.com/browse/MTV-603[(MTV-603)]
 
 .Snapshots that are created during a migration in OpenStack are not deleted
 
-In previous releases of {project-short}, the `Migration Controller` service did not delete snapshots that were created during a migration of source virtual machines in OpenStack automatically.
+In earlier releases of {project-short}, the `Migration Controller` service did not delete snapshots that were created during a migration of source virtual machines in OpenStack automatically.
 
 This issue is resolved in {project-short} {project-version}, all the snapshots created during the migration are removed after the migration has been completed. link:https://issues.redhat.com/browse/MTV-620[(MTV-620)]
 
 .{rhv-short} snapshots are not deleted after a successful migration
 
-In previous releases of {project-short}, the `Migration Controller` service did not delete snapshots automatically after a successful warm migration of a VM from {rhv-short}.
+In earlier releases of {project-short}, the `Migration Controller` service did not delete snapshots automatically after a successful warm migration of a VM from {rhv-short}.
 
 This issue is resolved in {project-short} {project-version}, the snapshots generated during migration are removed after a successful migration, and the original snapshots are not removed after a successful migration. link:https://issues.redhat.com/browse/MTV-349[(MTV-349)]
 
 .Warm migration fails when cutover conflicts with precopy
 
-In previous releases of {project-short}, the cutover operation failed when it was triggered while precopy was being performed. The VM was locked in {rhv-short} and therefore the `ovirt-engine` rejected the snapshot creation, or disk transfer, operation.
+In earlier releases of {project-short}, the cutover operation failed when it was triggered while precopy was being performed. The VM was locked in {rhv-short} and therefore the `ovirt-engine` rejected the snapshot creation, or disk transfer, operation.
 
 This issue is resolved in {project-short} {project-version}, the cutover operation is triggered, but it is not performed at that time because the VM is locked. Once the precopy operation completes, the cutover operation is triggered. link:https://issues.redhat.com/browse/MTV-686[(MTV-686)]
 
 .Warm migration fails when VM is locked
 
-In previous releases of {project-short}, triggering a warm migration while there was an ongoing operation in {rhv-short} that locked the VM caused the migration to fail because the snapshot creation could not be triggered.
+In earlier releases of {project-short}, triggering a warm migration while there was an ongoing operation in {rhv-short} that locked the VM caused the migration to fail because the it could not trigger the snapshot creation.
```

Comment:
Grammar: 
"In earlier releases of {project-short}, triggering a warm migration while there was an ongoing operation in {rhv-short} that locked the VM caused the migration to fail because **the it** could not trigger the snapshot creation."

Reply:
corrected